src/app/boards/boards.component.ts

```diff
@@ -34,7 +36,13 @@ export class BoardsComponent implements OnInit {
     editModal: false,
   };
 
-  currentUser: any;
+  status = {

```

Comment:
I don't like the name of 'StatusE'. I believe, that it could be simple 'Status'. And you don't need to create object and set values. Just - status = Status